.github/workflows/docs.yml

```diff
@@ -0,0 +1,34 @@
+name: Rust Docs
```

Comment:
Should we remove this workflow given that it will be taken care of by the same gitlab workflow for current docs?